arkane/common.py

```diff
@@ -268,19 +269,19 @@ def load_yaml(self, path, species, pdep=False):
                       'SingleExponentialDown': SingleExponentialDown,
                       'Wilhoit': Wilhoit,
                       'NASA': NASA,
+                      'NASAPolynomial': NASAPolynomial,
                       }
         freq_data = None
         if 'imaginary_frequency' in data:
             freq_data = data['imaginary_frequency']
-            del data['imaginary_frequency']
+            data = {key: data[key] for key in data.iterkeys() if key != 'imaginary_frequency'}
+        data['species'] = Species().fromSMILES(data['smiles'])
+        data['species'].label = data['label']
         self.make_object(data=data, class_dict=class_dict)
         if freq_data is not None:
             self.imaginary_frequency = ScalarQuantity()
             self.imaginary_frequency.make_object(data=freq_data, class_dict=dict())
-        self.adjacency_list = data['adjacency_list'] if 'adjacency_list' in data else None
-        self.inchi = data['inchi'] if 'inchi' in data else None
-        self.smiles = data['smiles'] if 'smiles' in data else None
-        self.is_ts = data['is_ts'] if 'is_ts' in data else False
```

Comment:
Are these no longer necessary?

Reply:
I don't believe these are necessary, but I could be wrong. I tested generating ArkaneSpecies yaml files, and everything looked okay (but I'll check again).

I believe this is the case because if these attributes were in `data` then these attributes would be passed to ArkaneSpecies.__init__() by `make_object`, and all 4 of these can be sent to __init__() and will be set there.

If there is a more subtle reason why these things were there though it is possible that they may still be needed.

Reply:
I think these are here just to report information in the YAML file, since Species doesn't have a .smiles .adjlist, .inchi, and .is_ts attributes. They shouldn't be processed when reading a YAML file, though. I think it's nice and convenient to have adjList / SMILES in the YAML file

Reply:
I am not sure I follow completely--let's talk offline about this

Reply:
@alongd and I discussed this offline. We ran an example Arkane job to confirm that these attributes are indeed stored on the YAML file even without these lines as expected.